tests/integration/run_tests.sh

```diff
@@ -74,10 +74,6 @@ run_test_case() {
         echo "${output}"
         echo "------ test output end ------"
 
-        echo "------ otbr log begin ------"
-        cat "${OTBR_LOG}"
-        echo "------ otbr log end ------"
-
```

Comment:
should we save and display ot-daemon log in case of test failure?

Reply:
Done.